kobo/apps/stripe/tests/test_mfa_login.py

```diff
@@ -212,12 +180,9 @@ def test_mfa_login_with_canceled_subscription_but_previously_set(self):
             'login': 'someuser',
             'password': 'someuser',
         }
-        response = self.client.post(
-            reverse('kobo_login'), data=data, follow=True
-        )
+        response = self.client.post(reverse('kobo_login'), data=data, follow=True)
         self._assert_mfa_login(response)
 
-    @override_config(MFA_ENABLED=True)
```

Comment:
Why did you remove this? 
We want to be sure it's enabled whatever is in the config, right?

That's the same comment for all deletions below ;-)